invenio_deposit/bundles.py

```diff
@@ -82,7 +82,9 @@
     'node_modules/invenio-records-js/dist/invenio-records-js.js',
     npm={
         'angular-schema-form': '~0.8.13',
-        'invenio-records-js': '0.0.6',
+        'invenio-records-js':
+            'git://github.com/galterlibrary/invenio-records-js'
+            '.git#bb4abf3513bfe6c2f771b2f840b6b2118cdbcec6',
```

Comment:
What if we added `our_master` branch on our `invenio-records-js`, and make that the default, would line 87 still be needed? This would eliminate the need to maintain this line then, right?

Reply:
Yes! I had to double check, but branch notation can indeed be used here. The integration branch can be placed here and we wouldn't need to worry about hashes in the future. I will wait for the other branch to be merged and then I will change this one.

Reply:
Great, one less thing to worry about.